ReText/window.py

```diff
@@ -324,13 +327,15 @@ def __init__(self, parent=None):
 		menuEdit.addSeparator()
 		menuEdit.addAction(self.actionViewHtml)
 		menuEdit.addAction(self.actionPreview)
-		menuEdit.addAction(self.actionInsertTable)
-		menuEdit.addAction(self.actionTableMode)
 		if ReTextFakeVimHandler:
 			menuEdit.addAction(self.actionFakeVimMode)
 		menuEdit.addSeparator()
 		menuEdit.addAction(self.actionFullScreen)
 		menuEdit.addAction(self.actionConfig)
+		menuInsert.addAction(self.actionInsertTable)
+		menuInsert.addAction(self.actionTableMode)
```

Comment:
I think that table mode has nothing to do with insertion, better leave it where it was.

And because of that, the Insert menu is becoming too small, so I would make it a submenu of Edit.

Reply:
Ok, right